.gitignore

```diff
@@ -13,4 +13,5 @@ buildNumber.properties
 *.iml
 *.DS_Store
 example-service/example-integration-openapi-spec/src/main/resources/lib
-.vscode
\ No newline at end of file
+.vscode
+test/k6/node_modules/*
```

Comment:
The transpired files could also be excluded, but I left them in so the tests could be run without needing to transpire the test files.